dns-controller/cmd/dns-controller/main.go

```diff
@@ -85,6 +88,18 @@ func main() {
 	flags.AddGoFlagSet(flag.CommandLine)
 	flags.Parse(os.Args)
 
+	var internalRecordTypes []dns.RecordType
+	if ipv4 {
```

Comment:
We could just default to ipv4.
```suggestion
	if ipv4 || !ipv6 {
```

Reply:
I think it'd be more worthwhile to catch cases where this information has been omitted.

Reply:
👍